opendebates/settings.py

```diff
@@ -8,6 +8,31 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 FIXTURE_DIRS = [os.path.join(BASE_DIR, 'fixtures')]
 
+SITE_ID = 1
+SITE_DOMAIN = os.environ.get("SITE_DOMAIN", "127.0.0.1:8000")
```

Comment:
There's not actually anything setting SITE_DOMAIN in our environment, so this always ends up as 127.0.1:8000.  

I'll look for something better.